main.🍇

```diff
@@ -30,6 +30,11 @@
     🔤t-rex🔤 ➡️ 🆕😶 🔤🦖🔤 🍿 🔤animal🔤 🍆 ❗️
     🔤slightly-smiling-face🔤 ➡️ 🆕😶 🔤🙂🔤 🍿 🔤face🔤 🔤smile🔤 🍆 ❗️
     🔤smirking-face🔤 ➡️ 🆕😶 🔤😏🔤 🍿 🔤face🔤 🔤smug🔤 🔤joke🔤 🍆 ❗️
+    🔤fire🔤 ➡️ 🆕😶 🔤🔥🔤 🍿 🔤fire🔤 🔤hot🔤 🔤heat🔤 🍆 ❗️
+    🔤snowflake🔤 ➡️ 🆕😶 🔤️❄️🔤 🍿 🔤ice🔤 🔤winter🔤 🔤snow🔤 🍆 ❗️
+    🔤party-face🔤 ➡️ 🆕😶 🔤️🥳🔤 🍿 🔤party🔤 🔤popper🔤 🔤celebration🔤 🍆 ❗️
+    🔤present🔤 ➡️ 🆕😶 🔤️🎁🔤 🍿 🔤present🔤 🔤gift🔤 🔤birthday🔤 🍆 ❗️
```

Comment:
```suggestion
    🔤present🔤 ➡️ 🆕😶 🔤️🎁🔤 🍿 🔤present🔤 🔤gift🔤 🔤birthday🔤 🔤holiday🔤 🔤christmas🔤 🍆 ❗️
```
Since gifts are exchanged on some holidays, and Christmas is one of those holidays, I think this makes sense. Are there any more holidays that you can think of?

Reply:
Thanksgiving , some Indian festivals like Diwali , Ramzan